### PR TITLE
Danya -- Hometask 26.09.21

### PR DESCRIPTION
**Pull request by ozgreat**

### CODE REVIEW

File: danya-homerask-27-09-21/1-max-consecutive-ones.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                cur++;
+            } else {
+                if (cur > max) {
+                    max = cur;
+                }
+                cur = 0;
+            }
+        }
+        
+        if (cur > max) {
```
Comment (BaLiKfromUA) [inline danya-homerask-27-09-21/1-max-consecutive-ones.java:17]: implement this task with **one** if statement, please

File: danya-homerask-27-09-21/12-move-zeroes.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Solution {
+    public void moveZeroes(int[] nums) {
+        int pointer = 0;
+        int zeroCount = 0;
```
Comment (BaLiKfromUA) [inline danya-homerask-27-09-21/12-move-zeroes.java:4]: IMO, you can implement this solution without `zeroCount`.
Try, please

File: danya-homerask-27-09-21/19-squares-of-a-sorted-array.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class Solution {
+    public int[] sortedSquares(int[] nums) {
+        for(int i = 0; i < nums.length; ++i) {
+            nums[i] = nums[i] * nums[i];
+        }
+        
+        Arrays.sort(nums);
```
Comment (BaLiKfromUA) [inline danya-homerask-27-09-21/19-squares-of-a-sorted-array.java:7]: Solution with sorting is very naive, try to solve this task with the two-pointers technique

File: danya-homerask-27-09-21/2-find-numbers-with-even-number-of-digits.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class Solution {
+    public int findNumbers(int[] nums) {
+        int even = 0;
+        
+        for(int i: nums) {
+            int digits = 0;
+            do {
```
Comment (BaLiKfromUA) [inline danya-homerask-27-09-21/2-find-numbers-with-even-number-of-digits.java:7]: count digits in a separate method, please

File: danya-homerask-27-09-21/3-duplicates-zeros.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+class Solution {
+    public void duplicateZeros(int[] arr) {
+        int zeroes = 0;
```
Comment (BaLiKfromUA) [inline danya-homerask-27-09-21/3-duplicates-zeros.java:3]: FOR WHAT do you need to count zeroes?))))

Comment (ozgreat) [reply danya-homerask-27-09-21/3-duplicates-zeros.java:3]: To get an increased array length which is the start point of the second poiner.

**Pull request merged**